packages/icons/scripts/icons.js

```diff
@@ -0,0 +1,35 @@
+const fs = require("fs");
+const path = require("path");
+
+const { pascalCase } = require("case-anything");
+const dotenv = require("dotenv");
+const { fetchImages } = require("figma-tools");
+const prettier = require("prettier");
+const svgtojsx = require("svg-to-jsx");
+
+dotenv.config({ path: "../../.env" });
+
+const pretterOptions = JSON.parse(fs.readFileSync("../../.prettierrc", "utf-8"));
+
+console.log("Fetching icons...");
```

Comment:
How long does the fetch step generally take?

Could add something like [progress-estimator](https://www.npmjs.com/package/progress-estimator) 😆  but I realize this is overkill

Reply:
Yeah, definitely! It's fairly quick right now, but it can slow as the Figma file grows over time and we add more. I had ora in there before for loading, but was having ESM issues 🙄. I'll circle back on this though and clean things up! My hope is we can start building a little design system for CLI tooling using react-ink.